src/parsing/syntax.ml

```diff
@@ -212,6 +213,18 @@ type p_tactic_aux =
 
 type p_tactic = p_tactic_aux loc
 
+type p_tactic_tree = Tactic of p_tactic option * p_tactic_tree list;;
+
+(**tree_to_list tactic_tree returns the list of tactics
+  described by the tree*)
+let rec tree_to_list : p_tactic_tree -> p_tactic list = function
+  | Tactic (otac, l) -> match otac with
```

Comment:
Thanks to try to follow the style of indentations used in other files. Here, match otac should be preceded by a newline.